pkg/actions/toolkit/enable.go

```diff
@@ -0,0 +1,123 @@
+package toolkit
+
+import (
+	"context"
+
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/flux"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
+)
+
+const allNamespaces = ""
+
+var defaultFluxComponents = []string{"helm-controller", "kustomize-controller", "notification-controller", "source-controller"}
+
+//go:generate counterfeiter -o fakes/fake_flux_client.go . FluxClient
+type FluxClient interface {
+	PreFlight() error
+	Bootstrap(opts *api.Toolkit) error
+}
+
+type Manager struct {
+	Opts         *api.Toolkit
+	K8sClientSet kubeclient.Interface
+	FluxClient   FluxClient
+}
+
+func New(k8sClientSet kubeclient.Interface, opts *api.Git) (*Manager, error) {
+	if opts.Toolkit == nil {
+		return nil, errors.New("expected git.toolkit in cluster configuration but found nil")
+	}
+
+	m := &Manager{
+		Opts:         opts.Toolkit,
+		K8sClientSet: k8sClientSet,
+		FluxClient:   flux.NewClient(opts.Toolkit.AuthTokenPath, opts.Toolkit.GitProvider),
+	}
+
+	return m, nil
+}
+
+func (m *Manager) Run() error {
```

Comment:
`toolkitManager.Run()` doesn't really describe what its doing 😉

Reply:
do you want a comment or a different func name?

Reply:
🤕  I think it could be updated to be more descriptive, atm I don't really understanding what it does from the func name

Reply:
it makes slightly more sense in `installer.Run()`

Reply:
`installer.Install()` gets me told off by the linter, as would `installer.InstallXComponents()` probably.

`installer.RunInstallation()`? I don't know if that is much better.

Reply:
`installer.Run` is good, before `manager.Run` sort of didn't really imply what was happening (what does telling the `manager` to `Run` do?), but I think telling the `installer` to `Run` shows its going to install 👌

Reply:
It was always meant to be `installer`, I just forgot to rename `manager` in that package 🙃